index.html

```diff
@@ -1,657 +1,1338 @@
-                    <div data-asset-id="" id="" data-cmp-data-layer="" class="cmp-image" itemscope="" itemtype="">
-                        <img srcset="./assets/alert-img.png" src="./assets/alert-img.png" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
+                <div class="container responsivegrid button-wrap">
+                  <div id="container-b090b78429" class="cmp-container">
+                    <div class="button">
+                      <a id="button-30f7ba2bf3" class="cmp-button" href="/content/leaderimpact/lang/en2/join-a-group.html">
+                        <span class="cmp-button__text">Join a Group</span>
+                      </a>
                     </div>
                     <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Learn More</span>
-                        </a>
+                      <a id="button-92e007d342" class="cmp-button" href="/content/leaderimpact/lang/en2/start-a-group.html">
+                        <span class="cmp-button__text">Start a Group</span>
+                      </a>
                     </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="welcome" style="background-image: url('./assets/Welcome-bg.png')">
-            <div class="container">
-                <div class="content-wrap">
-                    <div class="sub-title">
-                        <div data-cmp-data-layer="" id="" class="cmp-title">
-                            <h6>Welcome</h6>
-                        </div>
-                    </div>
-                    <div class="title">
-                        <div data-cmp-data-layer="" id="" class="title-wrap">
-                            <h3>Changing the World Takes Leaders Like You</h3>
-                        </div>
-                    </div>
-                    <div class="text">
-                        <div data-cmp-data-layer="" id="" class="text-wrap">
-                            <p>When leaders are transformed, we affect change through the channels and connections they have – family, friends, co-workers and colleagues. Deep, positive, long-lasting change that can help make our world better.</p>
-                        </div>
-                    </div>
-                    <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">More About Us</span>
-                        </a>
+          </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid alert-bar aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="container_alert" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 alert">

```

Comment:
This is one of the wrapping divs that needs to be removed, but the `alert` class can't go here.